definitions/npm/express_v4.x.x/flow_v0.32.x-/express_v4.x.x.js

```diff
@@ -0,0 +1,179 @@
+// @flow
+import type { Server } from 'http';
+
+declare type express$RouterOptions = {
+  caseSensitive?: boolean,
+  mergeParams?: boolean,
+  strict?: boolean
+};
+
+declare class express$RequestResponseBase {
+  app: express$Application;
+  get(field: string): string | void;
+}
+
+declare class express$Request extends http$IncomingMessage mixins express$RequestResponseBase {
+  baseUrl: string;
+  body: mixed;
+  cookies: {[cookie: string]: string};
+  fresh: boolean;
+  hostname: boolean;
+  ip: string;
+  ips: Array<string>;
+  method: string;
+  originalUrl: string;
+  params: {[param: string]: string};
+  path: string;
+  protocol: 'https' | 'http';
+  query: {[name: string]: string};
+  route: string;
+  secure: boolean;
+  signedCookies: {[signedCookie: string]: string};
+  stale: boolean;
+  subdomains: Array<string>;
+  xhr: boolean;
+  accepts(types: string): string | false;
+  acceptsCharsets(...charsets: Array<string>): string | false;
+  acceptsEncodings(...encoding: Array<string>): string | false;
+  acceptsLanguages(...lang: Array<string>): string | false;
+  header(field: string): string | void;
+  is(type: string): boolean;
+  param(name: string, defaultValue?: string): string | void;
+}
+
+declare type express$CookieOptions = {
+  domain?: string,
+  encode?: (value: string) => string,
+  expires?: Date,
+  httpOnly?: boolean,
+  maxAge?: number,
+  path?: string,
+  secure?: boolean,
+  signed?: boolean
+};
+
+declare type express$RenderCallback = (err: Error | null, html?: string) => mixed;
+
+declare type express$SendFileOptions = {
+  maxAge?: number,
+  root?: string,
+  lastModified?: boolean,
+  headers?: {[name: string]: string},
+  dotfiles?: 'allow' | 'deny' | 'ignore'
+};
+
+declare class express$Response extends http$ServerResponse mixins express$RequestResponseBase {
```

Comment:
This file is the same as the Flow v0.28.x - v0.31.x version, except for this line, where we make `express$Response` inherit from `http$ServerResponse`.

See also https://github.com/expressjs/express/blob/master/lib/response.js#L41